src/components/ui/OrganisationCard/index.tsx

```diff
@@ -45,6 +45,9 @@ export default function OrganisationCard(props: OrganisationCardProps): React.Re
 OrganisationCard.Wrapper = styled.div`
   box-shadow: 0px 1px 4px rgba(0, 0, 0, 0.1);
   border-radius: 3px;
+  margin: 5% auto;
+  width: 98%;
+  background-color: white;
 `;
 
 OrganisationCard.Content = styled.section`
```

Comment:
Please update this to a div tag for now.

Reply:
ok. right away

Reply:
done